### PR TITLE
[DOC-448] Document Sentiment Analysis [master]

### PR DESCRIPTION
**Pull request by ashish-macrometa**
- Updated documentation for Sentiment analysis  
- Updated example for Sentiment analysis 
- Fixes [[DOC-448](https://macrometa.atlassian.net/browse/DOC-448)]

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] Improvement (non-breaking change which improves existing functionality)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

[DOC-448]: https://macrometa.atlassian.net/browse/DOC-448?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by StoyanTc — APPROVED** [review]

File: docs/cep/reference/extensions/execution/sentiment.md [hunk 1]
```diff
@@ -1,5 +1,6 @@
 # Sentiment
-This extension performs sentiment analysis using Afinn Wordlist-based approach.
+This extension performs sentiment analysis using AFINN Wordlist-based approach. AFINN is a list of words rated for valence with an integer between minus five (negative) and plus five (positive).
+It contains 3300+ words with a polarity score associated with each word.
```
Comment (diana-macrometa) [inline docs/cep/reference/extensions/execution/sentiment.md]:
```suggestion
It contains more than 3,300 words with a polarity score associated with each word.
```

File: docs/cep/reference/extensions/execution/sentiment.md [hunk 2]
```diff
@@ -11,7 +12,7 @@ This extension performs sentiment analysis using Afinn Wordlist-based approach.
    
 ## getRate
 
-This provides the sentiment value for a given string as per the AFINN word list.
+This provides the sentiment value for a given string as per the AFINN word list. The sentiment value ranges from -5 to +5.
```
Comment (diana-macrometa) [inline docs/cep/reference/extensions/execution/sentiment.md]: Didn't we just say this on line 2? I suggest removing it in one place or the other.

File: docs/cep/reference/extensions/execution/sentiment.md [hunk 3]
```diff
@@ -11,7 +12,7 @@ This extension performs sentiment analysis using Afinn Wordlist-based approach.
...
 
 Syntax
```
Comment (diana-macrometa) [inline docs/cep/reference/extensions/execution/sentiment.md]:
```suggestion
## Syntax
```

**Review by diana-macrometa — CHANGES_REQUESTED** [review]
A few minor suggestions.

**Comment by diana-macrometa** [comment]
@ashish-macrometa, when this is ready for re-review, click the circling arrows next to my name. GitHub will send me a notification.

**Comment by diana-macrometa** [comment]
@ashish-macrometa, are you planning to finish this?

**Comment by diana-macrometa** [comment]
@ashish-macrometa, are you planning to finish this, or should we close the PR?

**Comment by diana-macrometa** [comment]
@ashish-macrometa or @StoyanTc, please finalize this topic. Respond to the suggestions and resolve conflicts.

**Review by StoyanTc — APPROVED** [review]

**Review by diana-macrometa — APPROVED** [review]

**Pull request closed without merging**